xcp_d/cli/run.py

```diff
@@ -74,7 +74,8 @@ def get_parser():
         "fmri_dir",
         action="store",
         type=Path,
-        help="the root folder of a preprocessed fMRI output .",
+        help="the root folder of a preprocessed fMRI output. If available, freesurfer outputs"
+        "should be added to this root folder, with the directory name containing 'freesurfer'",
```

Comment:
I don't think folks should move the freesurfer outputs to the fmri_dir. Rather, they should probably mount the freesurfer directory (or the directory containing both fmriprep and freesurfer derivatives) on the Docker/Singularity image.

Reply:
Instead, perhaps you could update the Docker/Singularity usage info. For example, the Docker call could look like this:

```
docker run --rm -it \
   -v /dset/derivatives/fmriprep:/fmriprep:ro \
   -v /tmp/wkdir:/work:rw \
   -v /dset/derivatives:/out:rw \
   -v /dset/derivatives/freesurfer:/freesurfer:ro \  # Necessary for fMRIPrep versions <22.0.2
   pennlinc/xcp_d:latest \
   /fmriprep /out participant \
   --cifti --despike --head_radius 40 -w /work --smoothing 6
```